### PR TITLE
Update sites.csv

### PR DESCRIPTION
**Pull request by melodykramer**
Added Carolina Public Press

### CODE REVIEW

File: newshomepages/sources/sites.csv [hunk 1]
```diff
@@ -75,3 +75,4 @@ BonginoReport,https://bonginoreport.com/,Bongino Report,,,,America/New_York,us-r
 BreitbartNews,https://www.breitbart.com/,Breitbart,,,,America/New_York,us-right-wing
 HumanEvents,https://humanevents.com/,Human Events,,,,America/New_York,us-right-wing
 OANN,https://www.oann.com/,One America News Network,,,,America/New_York,us-right-wing
+Carolina Public Press, https://carolinapublicpress.org/, Carolina Public Press,,,,America/New_York, north-carolina
```
Comment (palewire) [inline newshomepages/sources/sites.csv:78]:
```suggestion
Carolina Public Press,https://carolinapublicpress.org/,Carolina Public Press,,,,America/New_York,north-carolina
```
Let's cut these whitespaces.

**Review by palewire — CHANGES_REQUESTED** [review]

File: newshomepages/sources/sites.csv (hunk 1 above)
Comment (palewire) [inline newshomepages/sources/sites.csv:78]:
```suggestion
CPublicPress, https://carolinapublicpress.org/, Carolina Public Press,,,,America/New_York, north-carolina
```
This first field is intended to be the Twitter handle.

**Review by palewire — CHANGES_REQUESTED** [review]

File: newshomepages/sources/sites.csv (hunk 1 above)
Comment (melodykramer) [reply newshomepages/sources/sites.csv:78]: sounds good

Comment (melodykramer) [reply newshomepages/sources/sites.csv:78]: ah, sorry! I'll do the rest following this convention.

**Pull request merged**